ttnn/cpp/ttnn/operations/eltwise/op_a/device/kernels/dataflow/reader_unary_interleaved_start_id.cpp

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include "dataflow_api.h"
+#include "ttnn/cpp/ttnn/deprecated/tt_dnn/kernels/dataflow/moreh_common.hpp"
+
+#include "debug/dprint.h"
+
+void kernel_main() {
+    uint32_t src_addr  = get_arg_val<uint32_t>(0);
+    uint32_t num_tiles = get_arg_val<uint32_t>(1);
+    uint32_t start_id = get_arg_val<uint32_t>(2);
+    uint32_t param = get_arg_val<uint32_t>(3);
+
+    DPRINT_DATA0(DPRINT << "this is the data movement kernel on noc 0" << ENDL());
+    DPRINT << "testing " << ENDL();
+    DPRINT << "param is " << BF16(param) << ENDL();
+
+    constexpr bool src_is_dram = get_compile_time_arg_val(0) == 1;
+
+    constexpr uint32_t cb_id_in0 = 0;
+
+    // ublocks size defined in tiles
+    constexpr uint32_t onetile = 1;
+    const uint32_t tile_bytes = get_tile_size(cb_id_in0);
+    const DataFormat data_format = get_dataformat(cb_id_in0);
+
+    const InterleavedAddrGenFast<src_is_dram> s = {
+        .bank_base_address = src_addr,
+        .page_size = tile_bytes,
+        .data_format = data_format
+    };
+
+    // this fills the in0 CB with the param fill value
+    fill_cb_with_value(cb_id_in0, param);
```

Comment:
If I can fill the input CB with the `fill_value` parameter and copy this to output CB, can I simply perform this operation with just reader and writer kernel, without using eltwise unary sfpu compute kernel. if yes, how do I make that happen? The pack_tile for output CB is done in compute kernel file.